tests/ftests/071-sudo-pybindings-set_default_systemd_cgroup.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# set_default_systemd_cgroup functionality test using the python bindings
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli
+from libcgroup import Version, Cgroup
+import consts
+import ftests
+import sys
+import os
+
+
+CONTROLLER = 'cpu'
+SYSTEMD_CGNAME = '071_cg_in_scope'
+OTHER_CGNAME = '071_cg_not_in_scope'
+
+SLICE = 'libcgtests.slice'
+SCOPE = 'test071.scope'
+
+CONFIG_FILE_NAME = os.path.join(os.getcwd(), '071cgconfig.conf')
+SYSTEMD_DEFAULT_CGROUP_DIR = '/var/run/libcgroup'
+SYSTEMD_DEFAULT_CGROUP_FILE = '/var/run/libcgroup/systemd'
+
+# List of libcgroup.Cgroup objects
+CGRPS_LIST = []
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+
+    return result, cause
+
+
+def create_cgrp(config, CGNAME, controller=CONTROLLER, ignore_systemd=False):
+    global CGRPS_LIST
+
+    result = consts.TEST_PASSED
+    cause = None
+
+    cgrp = Cgroup(CGNAME, Version.CGROUP_V1)
+    if controller is not None:
+        cgrp.add_controller(controller)
+    cgrp.create()
+
+    if not CgroupCli.exists(config, controller, CGNAME, ignore_systemd=ignore_systemd):
+        result = consts.TEST_FAILED
+        cause = 'Failed to create /sys/fs/cgroup/{}/{}'.format(CONTROLLER, CGNAME)
+        return result, cause
+
+    CGRPS_LIST.append(cgrp)
+
+    return result, cause
+
+
+def setup(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if not os.path.isdir(SYSTEMD_DEFAULT_CGROUP_DIR):
+        os.mkdir(SYSTEMD_DEFAULT_CGROUP_DIR)
+
+    # Emulate the systemd slice/scope creation
+    f = open(SYSTEMD_DEFAULT_CGROUP_FILE, 'w')
+    f.write(os.path.join(SLICE, SCOPE))
+    f.close()
+
+    if not os.path.exists(SYSTEMD_DEFAULT_CGROUP_FILE):
+        result = consts.TEST_FAILED
+        cause = 'Failed to create %s' % SYSTEMD_DEFAULT_CGROUP_FILE
+        return result, cause
+
+    # create /sys/fs/cgroup/cpu/libcgtests.slice
```

Comment:
What if this test is run on a cgroup v2 system?  This comment isn't correct I think

Reply:
Sure, will replace this in the next version.